src/main/java/org/broadinstitute/hellbender/utils/downsampling/PassThroughDownsampler.java

```diff
@@ -0,0 +1,86 @@
+package org.broadinstitute.hellbender.utils.downsampling;
+
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * Pass-Through Downsampler: Implementation of the ReadsDownsampler interface that does no
+ * downsampling whatsoever, and instead simply "passes-through" all the reads it's given.
+ * Useful for situations where you want to disable downsampling, but still need to use
+ * the downsampler interface.
+ */
+public final class PassThroughDownsampler<T extends GATKRead> extends ReadsDownsampler<T> {
+
+    private LinkedList<T> selectedReads;
```

Comment:
Sigh...please create a ticket to eliminate misguided use of `LinkedLists` in the downsampling package, and assign to me.